Automattic/woocommerce-payments
# Track payment request button load events

#### Changes proposed in this Pull Request
Add Tracks events to record Apple Pay and Google Pay button rendering. 









#### Testing instructions




### Set-up
1. As a merchant, enable Google Pay/Apple Pay from the Payments -> Settings -> Express Checkouts
2. As a customer add a product to the cart

### Test the event
1. Visit the short code cart page
2. Make sure the Google Pay button is visible (You can also test for the Apple Pay button but you'd need to setup a live account) 
3. In ~5 minutes, check MC Tracks Live view for `wcpay_gpay_button_load` event with the `source: checkout` prop.
4. Repeat the above steps for the short-code checkout page, blocks cart, blocks checkout page, and product page. The event name should be the same but `source` property will match the relevant page (`checkout|cart|product`)

-------------------


**Post merge**
